core/src/main/java/com/datastax/oss/driver/api/core/uuid/Uuids.java

```diff
@@ -202,13 +235,188 @@ private static long makeClockSeqAndNode() {
   /**
    * Creates a new random (version 4) UUID.
    *
-   * <p>This method is just a convenience for {@code UUID.randomUUID()}.
+   * <p>This method is just a convenience for {@link UUID#randomUUID()}.
    */
   @NonNull
   public static UUID random() {
     return UUID.randomUUID();
   }
 
+  /**
+   * Creates a new name-based (version 3) {@link UUID} from the given namespace UUID and the given
+   * string representing the name part.
+   *
+   * <p>Note that the given string will be converted to bytes using {@link StandardCharsets#UTF_8}.
+   *
+   * @param namespace The namespace UUID to use; cannot be null.
+   * @param name The name part; cannot be null.
+   * @throws NullPointerException if <code>namespace</code> or <code>name</code> is null.
+   * @throws IllegalStateException if the {@link MessageDigest} algorithm for version 3 (MD5) is not
+   *     available on this platform.
+   */
+  @NonNull
+  public static UUID nameBased(@NonNull UUID namespace, @NonNull String name) {
+    Objects.requireNonNull(name, "name cannot be null");
+    return nameBased(namespace, name.getBytes(StandardCharsets.UTF_8));
```

Comment:
nit: `StandardCharsets.UTF_8` can be moved to constant

Reply:
It's already a constant, so why would you create another constant?

Reply:
I meant to create a field in this class and reference it, but you are right.